common/app/views/fragments/footer.scala.html

```diff
@@ -21,7 +23,7 @@
                     <span class="control__info">back to top</span>
                 </a>
             </div>
-            @if(showNav) {
+            @if(showNav && !amp) {
```

Comment:
ur correct - && is logical and, and & is binary and.  99% of the time you need &&